src/Carnac/App.xaml.cs

```diff
@@ -44,6 +56,30 @@ protected override void OnStartup(StartupEventArgs e)
             carnac = new KeysController(keyShowViewModel.Messages, messageProvider, new ConcurrencyService(), settingsProvider);
             carnac.Start();
 
+            Observable
+                .Timer(TimeSpan.FromMinutes(5))
+                .Subscribe(async x =>
+                {
+                    try
+                    {
+#if DEBUG
```

Comment:
What benefits to we get by testing the updating for debug builds? If it's fiddly and more hassle than it's worth I'd be tempted to just drop this and only enable it for release builds.